src/services/members/interfaces/member.ts

```diff
@@ -1,26 +1,10 @@
-// global
 import { Session } from '@fastify/secure-session';
-import { Actor } from '../../../interfaces/actor';
-import { UnknownExtra } from '../../../interfaces/extra';
+
+import { Member } from '@graasp/sdk';
 
 declare module 'fastify' {
   interface FastifyRequest {
```

Comment:
Would it make sense to also have this module augmentation in `@graasp/sdk`? Some plugins in separate repos use this knowledge that the request should have some auth information (e.g. websockets). Though it doesn't make sense if not all requests are necessarily authentified, this would confuse plugins that do not require sessions... Not sure what's best